spec/cypress/integration/admin/suppliers/edit-delivery-partner.js

```diff
@@ -1,67 +1,44 @@
 describe("Admin user editing delivery partner", () => {
+  const basePath = "/admin/suppliers";
   const leadProviderId = "e38e8825-4430-4da0-ac54-6e42dea5c360";
   const deliveryPartnerName = "Delivery Partner 1";
 
   beforeEach(() => {
     cy.login("admin");
   });
 
-  it("should update a delivery partner", () => {
+  it("should be able to edit a delivery partner", () => {
     cy.appScenario("admin/suppliers/manage_delivery_partner");
     const newName = "New delivery partner";
 
-    cy.visit("/admin/suppliers");
+    cy.visit(basePath);
     cy.get("a").contains(deliveryPartnerName).click();
 
-    cy.location("pathname").should(
-      "match",
-      /\/admin\/suppliers\/delivery-partners\/.*\/edit/
-    );
+    cy.location("pathname").should("match", /\/delivery-partners\/.*\/edit/);
     cy.get("[name='delivery_partner_form[name]']").should(
       "have.value",
       deliveryPartnerName
     );
     cy.get("[name='delivery_partner_form[name]']").type(
       `{selectall}${newName}`
     );
-    cy.clickCommitButton();
-
-    cy.location("pathname").should("equal", "/admin/suppliers");
-    cy.get("main").should("contain", newName);
-    cy.get("main").should("not.contain", deliveryPartnerName);
-  });
-
-  it("removes a lead provider when unchecked", () => {
-    cy.appScenario("admin/suppliers/manage_delivery_partner");
-
-    cy.visit("/admin/suppliers");
-
-    cy.get("a").contains(deliveryPartnerName).click();
-
     cy.get(
       `[name='delivery_partner_form[lead_provider_ids][]'][value=${leadProviderId}]`
     ).uncheck();
     cy.clickCommitButton();
 
-    cy.get("a").contains(deliveryPartnerName).click();
+    cy.location("pathname").should("equal", basePath);
```

Comment:
I do feel that cy.location("pathname") will be used a lot. Perhaps a helper method like current_path? Just a suggestion

Reply:
I prefer `cy.url()` and incomplete matching - but this is an area that needs more discussion (we could also go with titles, once we have them)